minimap2.wdl

```diff
@@ -0,0 +1,142 @@
+version 1.0
+
+# Copyright (c) 2019 Sequencing Analysis Support Core - Leiden University Medical Center
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+task Indexing {
+    input {
+        File referenceFile
+        String outputPrefix
+        Boolean useHomopolymerCompressedKmer = false
+        Int kmerSize = 15
+        Int minimizerWindowSize = 10
+
+        Int? splitIndex
+
+        Int cores = 1
+        Int memory = 4
+        String dockerImage = "quay.io/biocontainers/minimap2:2.17--h84994c4_0"
+    }
+
+    command {
+        set -e pipefail
```

Comment:
no pipes are used, so `pipefail` is not needed (and it should be `set -e -o pipefail` if it were needed)